defs/panda.def

```diff
@@ -168,6 +134,55 @@ void setXYZE(double px, double py, double pz, double e) override
   mass_ = std::sqrt(e * e - p * p);
 }
 
+% Note that SecondaryVertex stuff is copy-pasted from Vertex and RecoVertex
+% Multiple inheritance isn't working due to inheritance from Element
+[SecondaryVertex>ParticleM]
+x/F
+y/F
+z/F
+TVector3 position() const { return TVector3(x, y, z); }
+ntrk/s
+ndof/F
+chi2/F
+double normChi2() const { return ndof != 0. ? chi2 / ndof : chi2 * 1.e+6; }
+% pfRangeMax/s ??
+
+[PFCand>PackedParticle]
+packedPuppiW/B
+packedPuppiWNoLepDiff/B
+ptype/b
+vertex/RecoVertexRef/!
+track/PackedTrackRef/!
+secondaryVertex/SecondaryVertexRef
```

Comment:
Not all PF candidates have a secondary vertex ref, right? PF candidates is by far the largest collection, so each variable added to the object can increase the panda file size by a few percent. Can we instead save the indices of the associated candidates on the SecondaryVertex object and make this Ref dynamically set, like `vertex` and `track`? (ping me on Slack if you are not sure what I mean)

Reply:
So switching the reference to SecondaryVertex was easy enough. That's the model of CMSSW anyway, so that's probably what I should have followed in the first place. I guess I'm not sure about dynamically setting, but this seems like something that should be easy for people to work around.